woocommerce/woocommerce-android
# Fetching plugin if missing and block the button if fetching in progress

Closes: #6302


### Description

The PR adds fetching of the active plugin in cases when it's not cached. Also, it blocks a refund button on the time the network requests are running

### Testing instructions

* Clean install the app
* Try to refund interac payment
* Try to refund "normal"payment

To test the disabled button:
* Add delay in the code `PaymentChargeRepository::fetchCardDataUsedForOrderPayment`
* Notice that the button disabled and then gets enabled when delay has passed
